.github/workflows/CD.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: release to DockerHub
+on:
+  workflow_run:
+    workflows:
+      - CI
+    types:
+      - completed
+    branches:
+      - main
+    tags:
+      - '*'
+
+jobs:
+  build_docker_images:
+    if: github.event.workflow_run.conclusion == 'success'
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v3
+      - name: login to dockerhub
+        uses: docker/login-action@v1.14.1
+        with:
+          username: "${{ secrets.DOCKERHUB_USERNAME }}"
+          password: "${{ secrets.DOCKERHUB_TOKEN }}"
+          logout: false
+      - name: build docker-compose services
+        run: docker-compose build
+      - name: push docker image
+        run: docker-compose push
```

Comment:
Let's use [`build-push-action`](https://github.com/docker/build-push-action) to build production images and practice GitHub Actions a bit more ;)